epics-team-builder/src/widgets/SetupPage/components/ManuallyAssignProjects.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { Card, Table, Button, CardDeck } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+export default class ManuallyAssignProjects extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projectData: [{}],
+      studentData: [{}]
+    };
+  }
+
+  handleChangeProject(e) {
+    const id = e.target.id;
+
+    if (document.getElementById(id).checked) {
+      console.log('ID: ' + id);
+    }
+  }
+
+  handleChange = e => {
+    const id = e.target.id;
+    const value = e.target.value;
+    /* console.log(e.target.id) ;
+    console.log(document.getElementById(id).checked) ; */
+  };
+
+  handleClickStudent(e) {
+    const id = e.target.id;
+    var checkedStatus = document.getElementById(id).checked;
+    console.log(document.getElementById(id).checked);
+    /*  console.log(tempstudentIdArray[1]) ; */
+  }
+
+  addProjectToStudent(projects, students) {
+    var status = false;
```

Comment:
I don't see this variable being used, so we can just delete it if it won't be in the future. Running grunt shows a couple in this file being not being used, so just make sure to remove them if you have no plans for them anymore.